README.md

```diff
@@ -1,4 +1,4 @@
-![logo](Documentation/media/logo.png?raw=true "Rook")
+![logo](Documentation/media/logo.svg "Rook")
```

Comment:
Is there a way we can constrain the size of the image? It feels overwhelming.

Reply:
you guys don't know what you're talking about, I think it looks awesome!! 😵 

i'll see if it can be constrained somehow.

Reply:
OK i pushed an update to constraint it to 50% height and width.  I think it looks less cool now lol, but i think i'll probably be in the minority on that.  screenshots for reference included.  PTAL @travisn @galexrt 

## Rook logo 100%:
![Rook logo 100%](https://user-images.githubusercontent.com/4313439/56320700-8f6ef100-6119-11e9-9a1f-e46bc3b24c99.png)

## Rook logo 50%:
![Rook logo 50%](https://user-images.githubusercontent.com/4313439/56320699-8ed65a80-6119-11e9-8b31-cfa2eb7d9457.png)